target/linux/x86/64/config-6.6

```diff
@@ -151,7 +151,7 @@ CONFIG_DRM_FBDEV_EMULATION=y
 CONFIG_DRM_FBDEV_OVERALLOC=100
 CONFIG_DRM_GEM_SHMEM_HELPER=y
 # CONFIG_DRM_HYPERV is not set
-CONFIG_DRM_I915=y
+CONFIG_DRM_I915=m
```

Comment:
```suggestion
```
Please drop this since it's handled by video.mk.

Reply:
I mean just completely remove `CONFIG_DRM_I915` from Kconfig.

Reply:
I see, I can have a try. This is my first contribution to OpenWrt, are there any guidelines for this

Reply:
removed `COMFIG_DRM_I915` from `config-6.6` and it works as expected, patch is updated